src/webextension/manage/components/item-details.js

```diff
@@ -62,27 +61,42 @@ export default class ItemDetails extends React.Component {
         e.preventDefault();
         onSave(this.state);
       }}>
-        <p>
-          <Localized id="item-site">
-            <span>sITe</span>
+        <label>
+          <Localized id="item-details-title">
+            <span>tITLe</span>
           </Localized>
-          <Input type="text" value={this.state.title}
+          <Input type="text" name="title" value={this.state.title}
                  onChange={(e) => this.setState({title: e.target.value})}/>
-        </p>
-        <p>
-          <Localized id="item-username">
+        </label>
+        <label>
+          <Localized id="item-details-origin">
+            <span>oRIGIn</span>
+          </Localized>
+          <Input type="text" name="origin" value={this.state.origin}
+                 onChange={(e) => this.setState({origin: e.target.value})}/>
+        </label>
+        <label>
+          <Localized id="item-details-username">
             <span>uSERNAMe</span>
           </Localized>
-          <Input type="text" value={this.state.username}
+          <Input type="text" name="username" value={this.state.username}
                  onChange={(e) => this.setState({username: e.target.value})}/>
-        </p>
-        <p>
-          <Localized id="item-password">
+        </label>
+        <label>
+          <Localized id="item-details-password">
             <span>pASSWORd</span>
           </Localized>
-          <Input type="text" value={this.state.password}
+          <Input type="text" name="password" value={this.state.password}
```

Comment:
Should this be `input="text"` or `input="password"`. I was a bit surprised the first time I tried adding a new entry and my password was displayed in plaintext.
Not sure if we want to consider something like [**react-input-password**](https://www.npmjs.com/package/react-input-password) or [**react-password-mask**](https://www.npmjs.com/package/react-password-mask).

Reply:
It should be `input="text"`; hiding the password is #53.